pkgs/by-name/gh/ghostscript/package.nix

```
@@ -96,6 +96,8 @@ stdenv.mkDerivation rec {
   ;
 
   preConfigure = ''
+    export DARWIN_LDFLAGS_SO_PREFIX=$out/lib/
```

Assuming that the variable is needed for `autoconf` (otherwise it's more common to append at the end).

```suggestion
  preConfigure = ''
    ${lib.optionalString stdenv.hostPlatform.isDarwin ''
      export DARWIN_LDFLAGS_SO_PREFIX=$out/lib/
    ''}
```

or prepend it:

```
preConfigure= lib.optionalString stdenv.hostPlatform.isDarwin ''
      export DARWIN_LDFLAGS_SO_PREFIX=$out/lib/
    '' + ''
    [the rest]
```

---

Thanks, applied!